accounts/views.py

```diff
@@ -144,6 +144,13 @@ def complete_profile(request):
     # if user.desig == 'v':
     if request.method == 'POST':
         roll_no = request.POST['roll_no']
+        # To check whether entered roll number is already existed or not?
```

Comment:
Comments should be written in imperative form, like you are giving order to the following code to do something.

Writing "Check whether the entered roll number already exists." would be good here.